### DISCUSSION TITLE
dropgalaxy antiadb discussion

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockO/uBO-Scriptlets#30] [General] [answered]
@uBlock-user 
what we have advanced solutions which can deal with antiadblock on dropgalaxy(probably you are aware of history & **filter list maintainers already gave up**) => i am looking for solution which is privacy respecting=> see current fix in uassets, allowing adservers/trackers with antiadblock present & no one is bothers about allowing adservers 

`https://dgdrive.pro/llp5ame0crva`
try to download the file & in end you get antiadblock

**Comment by uBlock-user** [comment, marked as answer]
Despite having `@@*script,from=...` & `@@*$xhr,from=...`, I see anti-adblock.

No amount of advanced solutions will work when the issue turns into a war of attrition.

**Reply by ghajini** [reply]
1)what those filters doing anyway in ublock=> benefitting site admin & exposing users to tracking but filter list maintainers do not understand this point of respecting users privacy

2)at least can we start fixing antiadb on say weekly basis if attrition is too much?

**Reply by ghajini** [reply]
i tried `https://github.com/uBlockOrigin/uAssets/discussions/21766#discussioncomment-8013723`
but maintainer is busy & can't maintain

**Reply by uBlock-user** [reply]
> 2)at least can we start fixing antiadb on say weekly basis if attrition is too much?

Need to see the anti-adb script which I can't find.

**Reply by uBlock-user** [reply]
> https://github.com/uBlockOrigin/uAssets/discussions/21766#discussioncomment-8013723

He doesn't have any solution for dropgalaxy
